File: fintechplatform/src/main/java/com/dwaplatform/android/account/balance/db/Balance.java
package com.dwaplatform.android.account.balance.db;

import com.dwaplatform.android.db.PlatformDB;
import com.raizlabs.android.dbflow.annotation.Column;
import com.raizlabs.android.dbflow.annotation.PrimaryKey;
import com.raizlabs.android.dbflow.annotation.Table;
import com.raizlabs.android.dbflow.structure.BaseModel;

/**
 * Created by ingrid on 07/12/17.
 */

@Table(database = PlatformDB.class)
public class Balance extends BaseModel {
    @PrimaryKey
    public String id;
    @Column
    public Long amount;
    @Column
    public String currency;
    @Column
    public String accountId;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public Long getAmount() {
        return amount;
    }

    public void setAmount(Long amount) {
        this.amount = amount;
    }
}

File: fintechplatform-ui/src/main/java/com/fintechplatform/ui/account/balance/db/Balance.kt
package com.fintechplatform.ui.account.balance.db

import com.fintechplatform.ui.db.PlatformDB
import com.raizlabs.android.dbflow.annotation.Column
import com.raizlabs.android.dbflow.annotation.PrimaryKey
import com.raizlabs.android.dbflow.annotation.Table
import com.raizlabs.android.dbflow.structure.BaseModel

@Table(database = PlatformDB::class)
data class Balance(@PrimaryKey var id: String = "",
                   @Column var amount: Long? = null,
                   @Column var currency: String = "",
                   @Column var accountId: String = "",
                   @Column var availableBalance: Long? = null): BaseModel()